src/webapp/tests/component-tests/uploader/js/FileQueueViewTests.js

```diff
@@ -53,96 +53,103 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
-            index : 1, // The index of this file for use in getFile(i) 
-            name : "Ocean.jpg", // The file name. The path is not included. 
-            size : 950000000 // The file size in bytes        
+            id : 230948230984, // SWFUpload file id, used for starting or cancelling and upload
+            index : 1, // The index of this file for use in getFile(i)
+            name : "Ocean.jpg", // The file name. The path is not included.
+            size : 950000000 // The file size in bytes
         };
-                
+
         // Useful locate functions.
         var locateRows = function (q) {
-            return $("#main .flc-uploader-queue").find(q.options.selectors.fileRows);   
+            return $("#main .flc-uploader-queue").find(q.options.selectors.fileRows);
         };
-        
+
         var nameForRow = function (q, rowEl) {
             return rowEl.find(q.options.selectors.fileName).text();
         };
-        
+
         var sizeForRow = function (q, rowEl) {
             return rowEl.find(q.options.selectors.fileSize).text();
         };
-        
+
         var checkARIA = function (file, row) {
             jqUnit.assertEquals("The added row should have an aria-label attribute on it containing descriptive text about the file.",
                                 file.name + " " + fluid.uploader.formatFileSize(file.size), row.attr("aria-label"));
         };
-        
+
         // Reusable test functions
         var checkFileRow = function (q, file, row) {
             jqUnit.assertEquals("The added row should have the correct id attribute.",
                                 file.id, parseInt(row.prop("id"), 10));
             jqUnit.assertEquals("The added row should have the correct filename.",
                                 file.name, nameForRow(q, row));
             jqUnit.assertEquals("The added row should have the correct size.",
-                                fluid.uploader.formatFileSize(file.size), 
+                                fluid.uploader.formatFileSize(file.size),
                                 sizeForRow(q, row));
             checkARIA(file, row);
         };
-      
+
         var createFileQueue = function () {
             var uploader = fluid.initComponent("fluid.uploader.tests.multiFileUploader");
             return uploader.fileQueueView;
         };
-        
+
         // File Queue test case
         var setupFunction = function () {
             jqUnit.subvertAnimations();
         };
-        
+
         var fileQueueViewTests = new jqUnit.TestCase("FileQueueView Tests", setupFunction);
-        
+
+        fileQueueViewTests.test("Initialization", function () {
+             jqUnit.expect(2);
+             var q = createFileQueue();
+             jqUnit.assertNotUndefined("The fileQueueView is initialized", q);
+             jqUnit.assertEquals("The application role is added", "application", q.locate("fileQueue").attr("role"));
```

Comment:
Interestingly, this test currently fails, even through the role appears correctly in the DOM on the demo